src/assets/drizzle/scripts/drizzle.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+import 'prismjs';
+
+const dom = {};
+
+dom.nav = document.getElementById('nav');
+dom.navMenu = document.getElementById('nav-menu');
+dom.navToggle = dom.nav.querySelector('a[href="#nav"]');
+dom.navLinks = dom.navMenu.querySelectorAll('a');
+
+function setActiveNavItem (pathname) {
```

Comment:
I'll make an issue and more clearly describe what I mean. This wasn't the relevant place to initiate this discussion. My bad!